controller.js

```diff
@@ -1,7 +1,89 @@
-var Calculator = require( './Calculator' )
-var clickHandler = document.getElementById( 'calculator-keypad' )
+var Calc = new Calculator() //eslint-disable-line
+var clickElement = document.querySelector( '.calculator-keypad' )
 
-clickHandler.addEventListener( '.click', function( event ) {
-  var value = event.target
-  console.log( event.target )
-} )
+clickElement.addEventListener( 'click', calculatorHandler.bind( this ) )
```

Comment:
put all of this controller code into your Calculator object. Do the binding within the Constructor / initalization function.